lib/rsgem/tasks/create_gem.rb

```diff
@@ -32,6 +33,23 @@ def shell_command
           bundler_options
         ].compact.join(' ')
       end
+
+      def ensure_author
+        temp_username = 'change_me'
+        temp_email = 'change_me@notanemail.com'
+        gemspec = File.read(context.gemspec_path)
+        unless system('git config user.email')
+          puts "Warning: No git email set, setting #{temp_email} for now, please change this before publishing your gem."
+          gemspec.gsub!(/(spec.email[^\[]+)\[([^\]]+)/, "\\1\['#{temp_email}'")
```

Comment:
Do you think a simpler version of this regex would work, something like this maybe?

```suggestion
          gemspec.gsub!(/spec.email\s+=\s+.+/, "spec.email = '#{temp_email}'")
```

\s is used for the whitespaces. And we don't really care about maintaining the same amount of spaces in here (we are doing something similar here https://github.com/rootstrap/rsgem/blob/master/lib/rsgem/tasks/set_bundled_files.rb)

Reply:
Does this preserve the `[]` around `authors` and `email` that the gemspec defaults with?

Reply:
Oh you are right, I missed the brackets

```suggestion
          gemspec.gsub!(/spec.email\s+=\s+.+/, "spec.email = ['#{temp_email}']")
```